### ISSUE TITLE
Learning To Classify Images Without Label

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#47]
# TL;DR
In unsupervised classification, because it is difficult to make sense of clusters in k-means and, the authors propose a two-step process: 1. to make the distance of the representation closer to the augmented own data, and 2. to classify the nearest neighbor data into the same cluster. 83.5% Accuracy in CIFAR10.
![image](https://user-images.githubusercontent.com/24524018/90498934-dfcde700-e183-11ea-82d5-2c6287524f64.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2005.12320

# Submission Dates(yyyy/mm/dd)
2020/05/25

# Authors and institutions
Wouter Van Gansbeke, Simon Vandenhende, Stamatios Georgoulis, Marc Proesmans, Luc Van Gool
- Marc Proesmans
- Luc Van Gool 

# Methods

# Results

# Comments
ECCV2020
